diff --git a/setup.py b/setup.py
index 07c4034..8cae652 100644
--- a/setup.py
+++ b/setup.py
@@ -18,7 +18,7 @@ pkg_name= namespace + '.' + package
 wralea_name= namespace + '.' + package + '_wralea'
 
 
-meta_version = version 
+meta_version = version
 # check that meta version is updated
 f = pj(os.path.dirname(__file__),'src', 'openalea', 'lpy','__version__.py')
 d = {}
@@ -33,12 +33,13 @@ print pkg_name,': version =',version
 
 
 # Scons build directory
-build_prefix = "build-scons"
+#build_prefix = "build-scons"
 
 from setuptools import setup
-from openalea.deploy.binary_deps import binary_deps
+#from openalea.deploy.binary_deps import binary_deps
 
-install_requires = [binary_deps('vplants.plantgl')]
+#install_requires = [binary_deps('vplants.plantgl')]
+install_requires = []
 #if 'linux' not in sys.platform:
 #    install_requires.append('PyOpenGL')
 #    install_requires.append('pyqglviewer')
@@ -52,28 +53,28 @@ setup(
     author_email=authors_email,
     url=url,
     license=license,
-    
-    scons_scripts = ['SConstruct'],
-    
+
+    #scons_scripts = ['SConstruct'],
+
     namespace_packages = [namespace],
     create_namespaces = False,
-    
+
     # pure python  packages
     packages = [ pkg_name, pkg_name+'.gui',pkg_name+'.gui.plugins', pkg_name+'.cpfg_compat', wralea_name ],
-    py_modules = ['lpygui_postinstall'],
+    #py_modules = ['lpygui_postinstall'],
 
     # python packages directory
     package_dir = { '' : 'src',},
-                   
+
     # Add package platform libraries if any
     include_package_data = True,
     package_data = {'' : ['*.pyd', '*.so', '*.dylib', '*.lpy','*.ui','*.qrc'],},
     zip_safe = False,
 
     # Specific options of openalea.deploy
-    lib_dirs = {'lib' : pj(build_prefix, 'lib'),},
+    #lib_dirs = {'lib' : pj(build_prefix, 'lib'),},
     #bin_dirs = {'bin':  pj(build_prefix, 'bin'),},
-    inc_dirs = {'include' : pj(build_prefix, 'src','cpp') },
+    #inc_dirs = {'include' : pj(build_prefix, 'src','cpp') },
     share_dirs = {'share' : 'share', },
 
     # Dependencies
@@ -83,15 +84,15 @@ setup(
         'gui_scripts': ['lpy = openalea.lpy.gui.lpystudio:main',],
         'console_scripts': ['cpfg2lpy = openalea.lpy.cpfg_compat.cpfg2lpy:main',],
         },
-    
-    postinstall_scripts = ['lpygui_postinstall'],
-    
+
+    #postinstall_scripts = ['lpygui_postinstall'],
+
     # Dependencies
     setup_requires = ['openalea.deploy'],
-    dependency_links = ['http://openalea.gforge.inria.fr/pi'],
+    #dependency_links = ['http://openalea.gforge.inria.fr/pi'],
     install_requires = install_requires,
 
-    pylint_packages = ['src/openalea/lpy/gui']
+    #pylint_packages = ['src/openalea/lpy/gui']
     )
 
 
