diff --git a/QGLViewer/QGLViewer.pro b/QGLViewer/QGLViewer.pro
index 82746c2..f88d19b 100644
--- a/QGLViewer/QGLViewer.pro
+++ b/QGLViewer/QGLViewer.pro
@@ -212,7 +212,7 @@ unix {
 # -------------------
 macx|darwin-g++ {
 	# Default setting creates a Mac framework. Comment out this line to create a dylib instead.
-	!staticlib: CONFIG *= lib_bundle
+	#!staticlib: CONFIG *= lib_bundle
 
 	include.files *= qglviewer.icns
 
